appveyor.yml

```diff
@@ -32,6 +32,7 @@ only_commits:
     - sql/core/src/main/scala/org/apache/spark/sql/api/r/
     - core/src/main/scala/org/apache/spark/api/r/
     - mllib/src/main/scala/org/apache/spark/ml/r/
+    - core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala
```

Comment:
yes - it could?
that test in SparkSubmitSuite loads R file, if the test is changed there might not be a matching change in any R code to trigger test. I keep running in an earlier bug that SparkSubmitSuite is not matching the R files

Reply:
I see. Just wanted to double check. I am fine if it fixes a potential issue or worry as it won't harm anyway.